content/guides/models/track/runs/multiple-runs-per-process

```diff
@@ -0,0 +1,147 @@
+# Multiple Active Runs in One Process (Reinit Behavior)
```

Comment:
I believe we use "Sentence case" for all titles and headings.

How about,

> Simultaneous runs using `reinit="create_new"`

The title is shorter and informative enough that users who know the setting exists, but don't remember it, won't even have to click this entry in the navigation sidebar.